CHANGES.md

```diff
@@ -8,6 +8,7 @@ Next Release (5.11.0)
 Features
 --------
 * [#1398](https://github.com/java-native-access/jna/pull/1398): Increase `c.s.j.p.win32.Sspi#MAX_TOKEN_SIZE` on Windows 8/Server 2012 and later - [@dbwiddis](https://github.com/dbwiddis).
+* [#1401](https://github.com/java-native-access/jna/pull/1401): Added initial support for `oleacc.dll` and `IAccessible` interface - [@mobeigi](https://github.com/mobeigi).
```

Comment:
We tend to give the full path (abbreviated) to the class rather than a dll name, so `c.s.j.p.win32.Oleacc`.

Reply:
Ah awesome, I've update the entry!